tests/integration/kubernetes/gha-run.sh

```diff
@@ -424,6 +429,10 @@ function deploy_snapshotter() {
 }
 
 function cleanup_snapshotter() {
+	if [[ "${KATA_HYPERVISOR}" == "qemu-tdx" ]]; then
+	       echo "[Skip] ${SNAPSHOTTER} is pre-installed in the TEE machine"
```

Comment:
likewise

Reply:
When I added the log I mistakenly removed the return, sorry.